archimedes/src/app/component/program/program.component.html

```diff
@@ -1,6 +1,14 @@
-
-<li>
-    <a href="{{program.catalog_url}}" target="_blank">
-        {{program.name}}
-    </a>
-</li>
+<a href="{{program.catalog_url}}" *ngFor="let program of programs | slice:0:5" target="_blank" class="list-group-item">
```

Comment:
Not all programs will have a catalog URL--we'll need to render this as a `div` instead of a link in cases where the catalog URL isn't available.